src/harness/sas_interface.py

```diff
@@ -404,4 +404,44 @@ def assertValidResponseFormatForApprovedGrant(self, grant_response):
       Nothing. It asserts if something about the response is broken/not per
       specs. Assumes it is dealing with an approved request.
     """
-    pass
\ No newline at end of file
+    pass
+
+  @abc.abstractmethod
+  def assertRegistered(self, registration_request,
+                       conditional_registration_data=None):
+    """Register a list of devices.
+
+    Quickly register N devices, assert registration SUCCESS, get CBSD IDs.
+    Includes injection of FCC IDs and conditional registration data.
+
+    Args:
```

Comment:
Based on that, then we will only be using this Helper Function where we just need the Successful Registration before executing anything else? 

Because as mentioned in the comments by @harinineelu if we want to use this same assertRegistered like in the case of REG_14 #228 then it will get complicated because we have to make sure that we only call this assertion when the expectedResponseCode is 0.